src/main/java/org/sonarqube/A.java

```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
   public String foo() {
+    message = null;
+    message.length();
+
     return message;
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: A "NullPointerException" could be thrown; "message" is nullable here. ([AWBEzrmbI9pSNYZJmpeH](http://localhost:9000/coding_rules#rule_key=squid%3AS2259))
([See it in SonarQube](http://localhost:9000/project/issues?id=org.sonarqube%3Apr-decoration&branch=pr_28&issues=AWBEzrmbI9pSNYZJmpeH&open=AWBEzrmbI9pSNYZJmpeH)

[ISSUE_KEY]: AWBEzrmbI9pSNYZJmpeH